bin/dsn.templates/cpp/types.h.php

```diff
@@ -15,9 +16,6 @@
 //
 // !!! WARNING: not feasible for replicated service yet!!! 
 //
-// # define DSN_NOT_USE_DEFAULT_SERIALIZATION
-
-# ifdef DSN_NOT_USE_DEFAULT_SERIALIZATION
```

Comment:
In this file, rdsn will generate one of the three totally different codes according to serialization method specified by user before code generation, which makes this marco useless in this file.